app/services/template_kind_name_jar.rb

```diff
@@ -0,0 +1,21 @@
+class TemplateKindNameJar
+  def self.jar
+    @jar ||= { "PXELinux" => _("PXE Linux"),
+               "PXEGrub" => _("PXE Grub"),
+               "iPXE" => _("iPXE"),
+               "provision" => _("Provision"),
+               "finish" => _("Finish"),
+               "script" => _("Script"),
+               "user_data" => _("User Data"),
+               "ZTP" => _("ZTP"),
+               "POAP" => _("POAP")
```

Comment:
These will not get correctly translated, use `N_("...")` instead as there is no locale when creating this hash and the translations are evaluated..